{
  try {
    invoiceItemSqlDao.test(internalCallContext);
  }
 catch (  Throwable t) {
    Assert.fail(t.toString());
  }
}
